Relax/Relax/DataStore.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  DataStore.swift
+//  Relax
+//
+//  Created by Ryan Arana on 2/9/15.
+//  Copyright (c) 2015 PDX-iOS. All rights reserved.
+//
+
+import Foundation
+
+class DataStore {
```

Comment:
Another thing that might be useful is the concept of a "current activity"-- a way for the app to know the activity that's been started if the app is closed and re-launched before the user saves it.